lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -432,7 +432,9 @@
        (see L(here,https://github.com/docker/docker-py/issues/1310) for details).
        For Python 2.6, C(docker-py) must be used. Otherwise, it is recommended to
        install the C(docker) Python module. Note that both modules should I(not)
-       be installed at the same time."
+       be installed at the same time. Also note that when both modules are installed
```

Comment:
If:
1) This applies to all docker modules
2) All docker modules use the docs_fragment

THEN:
Maybe add to `lib/ansible/utils/module_docs_fragments/docker.py` rather than repeating in each module?

Reply:
@gundalow It does not apply to all docker modules; only to the ones which support `docker-py < 2.0`.

Reply:
Thanks for clarifying